client/src/main/scala/drt/client/services/handlers/CrunchUpdatesHandler.scala

```diff
@@ -124,11 +124,11 @@ class CrunchUpdatesHandler[M](airportConfigPot: () => Pot[AirportConfig],
         log.info(s"Got CrunchUpdates with ${cu.flights.size} flights, ${cu.minutes.size} minutes")
         UpdateCrunchStateFromUpdatesAndContinuePolling(cu)
       case _ =>
-        log.error(s"Failed to GetCrunchState. Re-requesting after $crunchUpdatesRequestFrequency")
+        log.info(s"Failed to GetCrunchState. Re-requesting after $crunchUpdatesRequestFrequency")
```

Comment:
Is this not still a bug?

Reply:
Is it a bug if there are no crunch updates?

Reply:
probably not. Maybe the message should be "No new CrunchUpdates, Re-requesting..."

Reply:
or do we need to handle the failure case separately to the "no new updates" case.